client/index.jsx

```diff
@@ -22,9 +23,9 @@ class App extends React.Component {
       type: 'GET',
       url: 'http://127.0.0.1:3002/games',
```

Comment:
You should simply use `/games` as the url option here. Referencing `localhost` or `127.0.0.1` will not work when you go to deploy this later to AWS or Heroku. As long as the client is connecting to the main Express server on Port 3002, that same hostname and port will be used for your API calls.

Reply:
implemented change

Reply:
implemented change